styles/index.css

```diff
@@ -1 +1,100 @@
 @import url("./something.css");
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 100%;
+  max-width: 1180px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.Header {
+}
+
+.Header-in {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 85px;
+}
+
+.Header-menu {
+  display: flex;
+  align-items: center;
+}
+
+.Header-left {
+}
+
+.Header-logo {
+}
+.Header-logo img {
```

Comment:
before a CSS rule add an empty line like you did for the previous ones.

Reply:
ok